src/test/java/example/tri/stringreverse/StringReverseTest.java

```diff
@@ -0,0 +1,13 @@
+package example.tri.stringreverse;
```

Comment:
You need to move this class to package tri.stringreverse (not example.tri.stringreverse). To do that, you go to IntelliJ, select your class StringReverseTest (in the left hand pane), right click, select Refactor > Move ..., then enter the new package new. Once you've done, it will change the package name, and automatically move your class to new folder: /src/test/java/tri/stringreverse/StringReverseTest.java

Why do we do this? Usually, the test class is in the same package as the real class. So you real class is tri.stringreverse.StringReverse, so your test class should be tri.stringreverse.StringReverseTest